gradle/changelog/square_bracket_encoding.yaml

```
@@ -0,0 +1,2 @@
+- type: added
```

shouldn't this be ...
```suggestion
- type: fixed
```